File: docs-site/ios/pub.swift

/*
* publish() returns a boolean
* Returns true if message was published
* Returns false if message was not published
*/

let success = try await realtime?.publish(topic: topic, message: [
    "user_name": "John Doe",
    "message": "How's it going fam?"
])

print("Message sent => \\(success)")

File: docs-site/android/pub.kt
val message = buildMap<String, Any> {
    put("user_name", "John Doe")
    put("message", "How's it going fam?")
}

/* 
* publish() returns a boolean
* Returns true if message was published
* Returns false if message was not published
*/
val sent = client.publish("chat.room1", message)

Log.d("Message", "Message sent => $sent")